src/components/vendor-tree.js

```diff
@@ -29,7 +31,7 @@ class VendorTree extends React.Component {
         {vendor}
       </span>
     );
-    const devices = this.props.treeData[vendor].map((item) => item);
+    const devices = _.filter(this.props.treeData, (item) => item.vendor === vendor).map((item) => item);
```

Comment:
to prevent filter on each render just group items by vendors and store it in `const` and then just get `grouppedItems[vendor]`

Reply:
fix eslint config to support line length 120 chars